service/src/test/java/bio/terra/cbas/dependencies/github/TestGithubService.java

```diff
@@ -0,0 +1,52 @@
+package bio.terra.cbas.dependencies.github;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class TestGithubService {
+
+  @Test
+  void returnCorrectBooleanValue() throws GitHubClient.GitHubClientException {
+    String jsonResponseFalse =
+        """
+        {
+          "name": "foo",
+          "organization": "broadinstitute",
+          "private": %s
+        }
+        """
+            .formatted("false")
+            .stripIndent()
+            .trim();
+
+    String jsonResponseTrue =
```

Comment:
Since they're so similar, I would make this a function like `public String jsonResponse(boolean isPrivate)` and just provide true/false as needed in the test cases